src/pudl/transform/eia861.py

```diff
@@ -2327,76 +2146,290 @@ def utility_data(tfr_dfs):
         how="outer",
     )
 
-    # Drop original operational_data_eia861 table from tfr_dfs
-    del tfr_dfs["utility_data_eia861"]
-
-    tfr_dfs["utility_data_nerc_eia861"] = transformed_ud_nerc
-    tfr_dfs["utility_data_rto_eia861"] = transformed_ud_rto
-    tfr_dfs["utility_data_misc_eia861"] = transformed_ud_misc
-
-    return tfr_dfs
+    return (
+        Output(
+            output_name="utility_data_nerc_eia861",
+            value=_post_process(transformed_ud_nerc),
+        ),
+        Output(
+            output_name="utility_data_rto_eia861",
+            value=_post_process(transformed_ud_rto),
+        ),
+        Output(
+            output_name="utility_data_misc_eia861",
+            value=_post_process(transformed_ud_misc),
+        ),
+    )
 
 
 ##############################################################################
-# Coordinating Transform Function
+# Secondary Transformations
 ##############################################################################
+@asset(
+    ins={
+        "advanced_metering_infrastructure_eia861": AssetIn(),
+        "demand_response_eia861": AssetIn(),
+        "demand_response_water_heater_eia861": AssetIn(),
+        "demand_side_management_ee_dr_eia861": AssetIn(),
+        "demand_side_management_misc_eia861": AssetIn(),
+        "demand_side_management_sales_eia861": AssetIn(),
+        "distributed_generation_fuel_eia861": AssetIn(),
+        "distributed_generation_misc_eia861": AssetIn(),
+        "distributed_generation_tech_eia861": AssetIn(),
+        "distribution_systems_eia861": AssetIn(),
+        "dynamic_pricing_eia861": AssetIn(),
+        "energy_efficiency_eia861": AssetIn(),
+        "green_pricing_eia861": AssetIn(),
+        "mergers_eia861": AssetIn(),
+        "net_metering_customer_fuel_class_eia861": AssetIn(),
+        "net_metering_misc_eia861": AssetIn(),
+        "non_net_metering_customer_fuel_class_eia861": AssetIn(),
+        "non_net_metering_misc_eia861": AssetIn(),
+        "operational_data_misc_eia861": AssetIn(),
+        "operational_data_revenue_eia861": AssetIn(),
+        "reliability_eia861": AssetIn(),
+        "sales_eia861": AssetIn(),
+        "utility_data_misc_eia861": AssetIn(),
+        "utility_data_nerc_eia861": AssetIn(),
+        "utility_data_rto_eia861": AssetIn(),
+    },
+)
+def utility_assn_eia861(**data_dfs: dict[str, pd.DataFrame]) -> pd.DataFrame:
```

Comment:
Moved this function down from above, to group it with the other "secondary transform" functions.